functions/Get-DbaSsisExecutionHistory.ps1

```diff
@@ -85,38 +92,65 @@ function Get-DbaSsisExecutionHistory {
         }
         if ($Status) {
             $csv = ($statuses[$Status] -join ',')
-            $statusq = "AND e.[Status] in ($csv)"
+            $statusq = "`n`t`tAND e.[Status] in ($csv)"
         }
         else {
             $statusq = ''
         }
-        
+
+        #construct parameterized collection predicate for project array
         if ($Project) {
-            $csv = "`"" + ($Project -join '","') + "`""
-            $projectq = "AND e.[ProjectName] in ($csv)"
+            $projectq = "`n`t`tAND ( 1=0 "
```

Comment:
We are not all that concerned with formatting in T-SQL....so no real reason to include all the tabs. The new line is added based on building the variable.

So if you have:
```powershell
"$var1
$var2
$var3
"
```
They will all be on a new line anyway.

Just an FYI....

Reply:
Pedantic's, ok if I keep it that way?

Reply:
And all that ugly white-space for parameters that have no value  ewwww.

Reply:
It will stay until someone cleans it up ;)